demos/iotivity/index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>OIC Tizen Crosswalk Extension demo</title>
+<link rel="stylesheet" type="text/css" href="css/iotivity.css"/>
+<meta name="viewport" content="width=device-width">
+<style>
+body {
+  font-size: 2em;
+}
+</style>
+</head>
+<body>
+
+    <div class="buttons-group" id="buttons-group">
+    <button class="SimpleServer" onclick="onclickbutton(id)" id="SimpleServer">SimpleServer</button>
+    <button class="SimpleClient" onclick="onclickbutton(id)" id="SimpleClient">SimpleClient</button>
+    </div>
+    <textarea cols=80 rows=25 id="output">i</textarea>
+
+</body>
+</html>
+
+
+<script>
+
+
+var outputtext = document.getElementById("output");
+outputtext.value = "";
+
+
+var log_outputtext = 1;
+
+
+function LogOutput(msg) {
+
+  if (log_outputtext)
+      outputtext.value += msg + "\n";
+  else
+    console.log(msg + "\n");
+}
+
+
+
+var lightResource = null;
+var device = new iotivity.OicDevice();
+
+device.configure({
+  url: "0.0.0.0:8453",
+  role: "intermediate",
+  connectionMode: "non-acked"
+  });
+
+var g_intervalId = 0;
+var g_intervalCount = 5;
+var g_observingCount = 5;
+var g_observedResource = null;
+
+
+function startServer() {
+  var deviceId = device.settings.info.uuid;
+  var connMode = device.settings.connectionMode;
+
+  var oicResourceInit = new iotivity.OicResourceInit({
```

Comment:
it seems to be this is only used in registerResource() - it would be more compact to directly use the property values from here in registerResource, and forget this variable.